wp-content/themes/access/views/programs/single.twig

```diff
@@ -178,7 +180,7 @@
           <meta itemprop="name" content="{{ post.government_agency }}" />
         </div>
 
-        {% if post.structured_data_service_type %}
+        {% if post.item_scope == 'GovernmentService' %}
```

Comment:
@theasteve I don't think this needed to change. Is there a reason you modified it? This wouldn't work the way it's written.

Reply:
@devowhippit `serviceType` and `serviceOperator` raised a warning:

> The property  is not recognized by Google for an object of type SpecialAnnouncement.

My understanding is that those properties shouldn't be render is the `item_scope` is `SpecialAnnouncement`. These should only be render when the item scope is `GovernmentService` since `item_scope` only returns `GovernmentService` or `SpecialAnnouncement`.

I can make a change to add a conditional `if post.item_scope != ' SpecialAnnouncement`

https://search.google.com/structured-data/testing-tool#url=https%3A%2F%2Faccessnyctest.wpengine.com%2Fprograms%2Fstimulus-benefit-coronavirus-bill%2F

<img width="911" alt="Screen Shot 2020-05-14 at 4 53 45 PM" src="https://user-images.githubusercontent.com/14843767/81989214-d83df580-960a-11ea-91d2-c7e366170724.png">

Reply:
I think you are right but I wouldn't worry about the warnings for now... they are because some of the GovernmentService items exist elsewhere on the page.

This block is already nested under a condition that prevents it from appearing if the item scope isn't SpecialAnnouncement (line 171). Another conditional against the itemscope type wouldn't be effective.

Reply:
Cool, in that case I will undo those changes.